packages/daemon/src/config.ts

```diff
@@ -49,6 +50,7 @@ export const FULLNODE_PEER_ID = process.env.FULLNODE_PEER_ID;
 export const FULLNODE_HOST = process.env.FULLNODE_HOST;
 export const STREAM_ID = process.env.STREAM_ID;
 export const NETWORK = process.env.NETWORK;
+export const FULLNODE_NETWORK = process.env.FULLNODE_NETWORK;
```

Comment:
Maybe we should add a comment explaining the reason why we needed 2 env vars (NETWORK and FULLNODE_NETWORK)

Reply:
Done! Thanks